Exam28/Exam28/Program.cs

```diff
@@ -0,0 +1,67 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Exam28
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            int maxNum = Convert.ToInt32("11111111111111111111111111111111", 2);
```

Comment:
Этот оператор дает `maxNum = -1`.
Здесь лучше использовать целые без знака
```suggestion
            uint maxNum = Convert.ToUint32("11111111111111111111111111111111", 2);
```